docs/sphinx_doc/en/source/tutorial/208-distribute.md

```diff
@@ -79,13 +81,20 @@ server.launch()
 server.wait_until_terminate()
 ```
 
-And run the following code on `Machine2`:
+> For similarity, you can run the following command in your terminal rather than the above code:
+>
+> ```shell
+> as_server --host ip_a --port 12001 --model-config-path model_config_path_a
```

Comment:
A little confused that why an agent server requires model configs? 
Shouldn't the model config be created and specified from the center node?